src/cli/init.ts

```diff
@@ -81,10 +82,49 @@ export function builder(yargs: Argv) {
 }
 export async function handler(argv: Arguments) {
   let baseDir = argv.dir as string;
+  let packageInstalled: boolean = false;
   let ts = argv.typescript;
   let updateTsConfig = argv.tsconfig;
   let changes: [Logger, string][] = [];
 
+  try {
+    packageInstalled = await checkPackageJsonForNextVideo('./package.json');
+  } catch (err: any) {
+    if (err.code === 'ENOENT') {
+      log.error(
+        `Failed to find/read a local package.json. Double check that you're running this from the root of your project.`
+      );
+      return;
+    }
+
+    console.log(err);
+  }
+
+  if (!packageInstalled) {
+    const install = await confirm({
+      message: `It doesn't look like ${chalk.magenta.bold(
+        'next-video'
+      )} is installed in this project. Would you like to install it now?`,
```

Comment:
Why ask? I don't think the sentry one does.

Reply:
I dunno, I'm a bigger fan of explicit choices generally, especially when we're mucking around in someone's project files. Principle of least surprise and all that.